src/components/v2/Layout/Header/CoinInfo.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import commaNumber from 'comma-number';
+import Typography from '@mui/material/Typography';
+import { Paper } from '@mui/material';
+import { useMarketsUser } from 'hooks/useMarketsUser';
+import { useVaiUser } from 'hooks/useVaiUser';
+import { Icon } from '../../Icon';
+import { useStyles } from './styles';
+
+const format = commaNumber.bindWith(',', '.');
```

Comment:
I feel this will be used in a lot of places. How about making it a reusable util?

Reply:
I'll open a PR for this